scripts/recombination/run.py

```diff
@@ -126,7 +126,7 @@ def gcloud_describe(operation_id, key_file):
 # Set ripples job config options
 #docker_image = "mrkylesmith/ripples_pipeline:latest"
 #TODO: Using dev image at the moment
-docker_image = "mrkylesmith/ripples_pipeline_dev:latest"
+docker_image = "yecheng/ripples_pipeline_dev:latest"
```

Comment:
I am testing using my container for now. I haven't found a solution for avoiding hardcoding docker image path.